data/json/items/vehicle_parts.json

```diff
@@ -170,6 +170,21 @@
         "cutting" : 0,
         "category" : "veh_parts",
         "price" : 20000
+    },{
+        "type":"GENERIC",
+        "id" : "w_table",
```

Comment:
OK, and can this be installed in vehicles?  I'm thinking this needs entries in the veh_part JSONs.